### ISSUE TITLE
[v3] Add tags

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/scala#651]
_This issue is part of the migration to v3. You can read full details about the various changes [here](https://github.com/exercism/v3-launch/)._

In Exercism v3, tracks can be annotated with tags. This allows searching for tracks with a certain tag combination, making it easy for students to find an interesting track to join.

Tags are specified in the top-level `"tags"` field in the track's `config.json` file and are defined as an array of strings, as specified in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#tags).

## Goal

The `"tags"` field in the `config.json` file should be updated to contain the tags that are relevant to this track. The list of tags that can be used is listed in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#tags).

## Example

```json
{
  "tags": [
    "runtime/jvm",
    "platform/windows",
    "platform/linux",
    "paradigm/declarative",
    "paradigm/functional",
    "paradigm/object_oriented"
  ]
}
```

## Tracking

https://github.com/exercism/v3-launch/issues/1

**Comment by MeerKatDev** [comment]
I would propose the following tags:
```
{
  "tags": [
    "runtime/jvm",
    "platform/windows",
    "platform/linux",
    "platform/mac",
    "platform/web",
    "paradigm/imperative",
    "paradigm/functional",
    "typing/static",
    "typing/dynamic",
    "typing/strong",
    "execution_mode/compiled",
    "used_for/backends",
    "used_for/frontends",
    "used_for/financial_systems"
  ]
}
```
Some points about the above:

- I added `platform/web` because Scala can be transpiled to javascript with Scala.js
- There are some other uses, but are very specific, I think someone else should complete this
- I added `typing/dynamic` because Scala can guess the type of a variable by what you assign.

**Comment by ErikSchierboom** [comment]
Scala can also be used to do object-oriented programming, right?

**Comment by MeerKatDev** [comment]
> Scala can also be used to do object-oriented programming, right?

Of course! Sorry, I don't know how I missed it.

**Comment by ErikSchierboom** [comment]
No worries

**Comment by ErikSchierboom** [comment]
Closed by #682
